cfme/tests/containers/test_provider_configuration_menu.py

```diff
@@ -17,10 +19,16 @@
 
 
 def select_first_provider_and_get_its_name():
-    navigate_to(ContainersProvider, 'All')
+    """Selecting the first provider and return its name.
+    If no such item found - return None."""
+    view = navigate_to(ContainersProvider, 'All')
     tb.select('Grid View')
-    Quadicon.select_first_quad()
-    return Quadicon.get_first_quad_title()
+    try:
+        entity = view.entities.get_first_entity()
+    except ItemNotFound:
+        return
+    entity.check()
```

Comment:
I don't recall whether selection is restored after navigation.
Please do paginator.check_all()/uncheck_all() if items are selected after navigation.

Reply:
This is already done here - https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/provider/__init__.py#L274.